src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -3809,20 +3804,21 @@ namespace CondenserLoopTowers {
                                                         RoutineName);
 
                 // full speed fan tower UA
-                Par[0] = tmpNomTowerCap * this->HeatRejectCapNomCapSizingRatio;
-                Par[1] = double(this->thisTowerNum);
-                Par[2] = rho * tmpDesignWaterFlowRate; // design water mass flow rate
-                Par[3] = tmpDesignAirFlowRate;         // design air volume flow rate
-                Par[4] = Cp;
-                UA0 = 0.0001 * Par[0]; // Assume deltaT = 10000K (limit)
-                UA1 = Par[0];          // Assume deltaT = 1K
+                Real64 const solveLoad = tmpNomTowerCap * this->HeatRejectCapNomCapSizingRatio;
+                Real64 const solveWaterFlow = rho * tmpDesignWaterFlowRate; // design water mass flow rate
```

Comment:
When the `Par` array was populated with expressions, I sometimes created intermediate variables that can be captured into the lambda.